src/python/pants/core/goals/lint.py

```diff
@@ -188,16 +195,15 @@ def register_options(cls, register) -> None:
                 "Specifying a directory causes linters that support writing report files to write "
                 "into this directory."
             ),
+            removal_version="2.7.0.dev0",
+            removal_hint=f"Edit the linter's config to cause it to write reports under "
+            f"{LINTER_REPORT_DIR} .",
```

Comment:
An example will go a long way:

```
e.g. add `--output={LINTER_REPORT_DIR}` to `[flake8].args` or to your Flake8 config file. 
```

Right now, it's not clear if you mean the tool's own config vs. Pants's config for the tool.